### ISSUE TITLE
Anchor text for screen readers

### ISSUE THREAD

**Issue by koalie** [w3c/w3c-website#693]
**To reproduce**
https://www.w3.org/sponsor/ 

**Describe the issue**
Relayed from @daniel-montalvo : 

Ideally anchor text for screen readers should include the text of the anchor. 
For example, <a href="#web-for-all-w3c-principles-champions" class="heading-anchor"><span aria-hidden="true">§</span><span class="visuallyhidden">anchor</span></a> 
should be <a href="#web-for-all-w3c-principles-champions" class="heading-anchor"><span aria-hidden="true">§</span><span class="visuallyhidden">anchor Web for All - W3C Principles Champions </span></a>. 

It's true that this is a common pattern that we are using (I've seen anchors like this on the calendar and probably other pages that I just can't remember now). This is AAA but still a huge improvement in usability.
